ystockquote.py

```diff
@@ -13,488 +13,497 @@
 #  Requires: Python 2.7/3.3+
 
 
-__version__ = '0.2.5dev'
+__version__ = '0.2.6dev'
 
 try:
-    # py3
-    from urllib.request import Request, urlopen
-    from urllib.parse import urlencode
+	# py3
+	from urllib.request import Request, urlopen
+	from urllib.parse import urlencode
+	import datetime
 except ImportError:
-    # py2
-    from urllib2 import Request, urlopen
-    from urllib import urlencode
-
+	# py2
+	from urllib2 import Request, urlopen
+	from urllib import urlencode
+	import datetime
```

Comment:
move this up so imports are alphabetized.

I think the correct order there should be:

```python
from urllib import urlencode
from urllib2 import Request, urlopen
import datetime
```